code/game/objects/items/storage/firstaid.dm

```diff
@@ -318,12 +318,21 @@
 /obj/item/storage/pill_bottle
 	name = "pill bottle"
 	desc = "It's an airtight container for storing medication."
-	icon_state = "pill_canister"
+	icon_state = "pill_canister_0"
 	icon = 'icons/obj/chemical.dmi'
 	item_state = "contsolid"
 	lefthand_file = 'icons/mob/inhands/equipment/medical_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
 	w_class = WEIGHT_CLASS_SMALL
+	var/pill_variance = 100 //probability pill_bottle has a different icon state. Put at 0 for no variance
+	var/pill_type = "pill_canister_"
+
+/obj/item/storage/pill_bottle/Initialize()
+	var/proper_name = name
+	. = ..()
+	name = proper_name
```

Comment:
What is the point of this name stuff?

Reply:
Won't this change the name of the icon for what the icon become ? I saw that being used somewhere else too

Reply:
Name and icon are independant, shouldn't affect pillbottles. What other thing was it on?

Reply:
Okay, that's what I thought too but I preferred to do as others. It was also used in the asteroid floor tile: 
![image](https://user-images.githubusercontent.com/69859497/103378032-55a66f80-4ae1-11eb-9917-a2a974e8e3c3.png)